fbchat/client.py

```diff
@@ -11,7 +11,12 @@
 from .models import *
 from .graphql import *
 import time
-
+try:
+    from urllib.parse import urlparse
+    from urllib.parse import parse_qs
+except ImportError:
+    from urlparse import urlparse
+    from urlparse import parse_qs
```

Comment:
When importing two things from the same module you can use commas, like this: `from urlparse import urlparse, parse_qs`